### DISCUSSION TITLE
GitHub Pages setting from account used for project

### DISCUSSION THREAD

**Discussion by willfaught** [community/community#32271] [Actions] [answered]
In the GitHub Pages config file `.github/workflows/pages.yml` (generated by GitHub for a Hugo GitHub Pages site), the value `${{ steps.deployment.outputs.page_URL }}` is getting the custom domain I used for my account's GitHub Pages, rather than the expected `account.github.io/repo` value for the project repo. I double checked, and no custom domain is set for the project repo.

**Comment by airtower-luna** [comment, marked as answer]
That is [documented behavior](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages):

> After you configure a custom domain for a user or organization site, the custom domain will replace the `<user>.github.io` or `<organization>.github.io` portion of the URL for any project sites owned by the account that do not have a custom domain configured.

I suppose renaming the user site repository should help, and with the custom domain it shouldn't matter otherwise if it's technically a "user site".

**Reply by willfaught** [reply]
Thanks!
